hack/istio/install-testing-demos.sh

```diff
@@ -125,6 +125,9 @@ if [ "${DELETE_DEMOS}" != "true" ]; then
   if [ "${USE_GATEWAY_API}" == "true" ]; then
     gateway_yaml="${ISTIO_DIR}/samples/bookinfo/gateway-api/bookinfo-gateway.yaml"
   elif ! [ -z "$GATEWAY_HOST" ]; then
+    # TODO: Ideally this wouldn't go in istio-system. It'd either go in the same namespace as the app
```

Comment:
Would it be a lot of work to refactor cypress test suite? ( only namespace and gateway name will change so ideally only names should be changed. or is there anything else ? )

Reply:
We discussed this offline but some of the frontend tests do rely on things like a certain number of objects in a table for a given namespace, the actual `Gateway` and `VirtualService` resource being present, etc. so it's probably more work than doing a find/replace.

Reply:
yes :+1:  , we can try to move istio-gateway to bookinfo namespace as a part of different namespace